travis-ci/vmtest/mkrootfs.sh

```diff
@@ -123,20 +115,34 @@ mkdir -m 755 "$root/etc/init.d" "$root/etc/rcS.d"
 cat > "$root/etc/rcS.d/S10-mount" << "EOF"
 #!/bin/sh
 
-/bin/mount -a
+set -eux
+
+/bin/mount proc /proc -t proc
+# Try mounting devtmpfs
+/bin/mount devtmpfs /dev -t devtmpfs || true
```

Comment:
why `|| true` here? is it expected to fail?

Reply:
To address the case when `devtmpfs` is mounted prior to `S10-mount` run:
```
+ '[' -x /etc/rcS.d/S10-mount ']'
+ /etc/rcS.d/S10-mount
+ /bin/mount proc /proc -t proc
+ /bin/mount devtmpfs /dev -t devtmpfs
mount: mounting devtmpfs on /dev failed: Resource busy
```

Reply:
I'll do a check before mounting instead of silencing with `|| true`.